File: examples/macro_drop/hump_python.cpp
#include <erCv/erCv.hpp>

int main( int hola, char** file_name)
{
  CvRect rect;
  rect.x=121;rect.y=110;rect.width=814;rect.height=223;
  erCerc cerc(135,293,25);
  erEqualP equ(0);
  erSmootP p1(BLUR,7),p2(MEDIAN,5);
  erCannyP cann(355,355);
  erAdThrP adp(THRESH_BINARY,AM_MEAN,49,119,255); //** < */
  erMacroDropAnalysis mda("hump");
  mda.defineParameters(rect,cerc,p1,p2,cann,adp,equ);
  mda.doIt("hump_1.bmp");
  return(0);
};

File: examples/macro_drop/hump_python.py
import sys
sys.path.append('../../libs')
from PyerCv import *

rect=CvRect();
rect.x =121;rect.y=110;rect.width=814;rect.height=220;
cerc= erCerc(135,293,25);
p1  = erSmootP(BLUR,7)
p2  = erSmootP(MEDIAN,5);
cann= erCannyP(355,355);
adp = erAdThrP(THRESH_BINARY,AM_MEAN,49,119,255);

an=erMacroDropAnalysis("hump","hump")
an.defineParameters(rect,cerc,p1,p2,cann,adp)
an.doIt("hump_1.bmp")
an.doIt("hump_2.bmp")
an.doIt("hump_3.bmp")
